gstcefsrc.cc

```diff
@@ -1111,6 +1257,10 @@ gst_cef_src_class_init (GstCefSrcClass * klass)
     g_param_spec_boolean ("sandbox", "sandbox",
           "Toggle chromium sandboxing capabilities",
           DEFAULT_SANDBOX, (GParamFlags) (G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS | GST_PARAM_MUTABLE_READY)));
+  g_object_class_install_property (gobject_class, PROP_LISTEN_FOR_JS_SIGNAL,
```

Comment:
Let's add a docstring here, and link to a section in the README explaining in detail the expected protocol between the element and the web page :)